### DISCUSSION TITLE
Getting detected with seleniumbase

### DISCUSSION THREAD

**Discussion by cym0n7** [seleniumbase/SeleniumBase#2355] [General]
As soon as I start a browser Session with the seleniumbase chromedriver and try to enter Credit Card Credentials to buy a item on a certein site, it wont let be buy the item and the website says "Cannot connect to Captcha service". All of this happens inside of an iFrame of this website. It seems to be hidden captcha, because normally you dont need to check a field or anything else when you do this. 

![screenie](https://github.com/seleniumbase/SeleniumBase/assets/116563356/66a96699-fcdf-41f8-bcbf-d221f361b730)

I  do pass any other bot protections like https://nowsecure.nl/#relax when using seleniumbase. 

When I enter the Credit Card Informations and buy the Item on my normal Browser without seleniumbase, it works with no problem. Same PC, same ISP, same Chrome Profile. 

This is my current code:
___
from seleniumbase import Driver

driver = Driver(uc=True, undetectable=True, undetected=True, agent="Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36", user_data_dir="C:/Users/Admin/AppData/Local/Google/Chrome/User Data")

driver.open(">linktowebsite>")
___

Do you guys know any arguments I could add or bypasses of Bot Protection I could try to prevent this? Thank you in advance.

**Comment by mdmintz** [comment]
Since you're doing more than just navigating to a website, try the newer UC Mode examples where clicking is required:
* https://github.com/seleniumbase/SeleniumBase/blob/master/examples/raw_turnstile.py
* https://github.com/seleniumbase/SeleniumBase/blob/master/examples/raw_form_turnstile.py

Also, you may need to use `driver.uc_open_with_reconnect(url, reconnect_time)`. Eg:

```python
from seleniumbase import Driver

driver = Driver(uc=True)
driver.uc_open_with_reconnect(URL, reconnect_time=5)
# ...
driver.quit()
```

See https://www.youtube.com/watch?v=5dMFI3e85ig for all details.
